monitor/src/api/hugin.js

```diff
@@ -1,6 +1,87 @@
-const endpoint = "http://localhost:8080/v1/statistics";
+const endpoint = "https://stage.xkr.mjovanc.com/api/v1/statistics/";
```

Comment:
I think we can use the `http://localhost:8080/api/v1/statistics` since it runs on the same server.

Reply:
hmm no becuase this will be called from clients aswell